pages/common/comm.md

```diff
@@ -0,0 +1,24 @@
+# comm
+
+> Select or reject lines common to two files. Both files must be sorted and share collation.
+
+- Produce three tab separated columns: lines only in first file, lines only in second file and common lines.
```

Comment:
No, sorry for not being clear :) I meant this could be "tab-separated" rather than "tab separated" (I believe that's the correct way in English)